lib/cfndsl/aws_types.yaml

```diff
@@ -939,6 +939,37 @@ Resources:
     Attributes:
      Arn: String
      QueueName: String
+  "AWS::WAF::ByteMatchSet" :
```

Comment:
Why do not we try to rebase with stevenjack/cfndsl? I think , these are already done in the parent repo.

Reply:
agreed, was going to discuss in standup, JIRA DO-2306 raised